src/ExpressionLanguage/ExpressionFunction/Security/HasAnyPermission.php

```diff
@@ -6,17 +6,25 @@
 
 use Overblog\GraphQLBundle\ExpressionLanguage\ExpressionFunction;
 use Overblog\GraphQLBundle\Generator\TypeGenerator;
+use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
 
 final class HasAnyPermission extends ExpressionFunction
 {
-    public function __construct($name = 'hasAnyPermission')
+    public function __construct(AuthorizationCheckerInterface $authorizationChecker, $name = 'hasAnyPermission')
```

Comment:
why keeping `$name` argument sometime?

Reply:
I initially planned to keep the `$name` variable only for classes with alias (`mutation`, `resolver`, `parameter` and `service`). I just forgot to remove them from other classes.

Fixed that.